include/concepts/View.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef KOKKOS_UTILS_CONCEPTS_VIEW_HPP
```

Comment:
I wonder if we should repeat the name of the repository as a subdirectory of `include`. E.g., we'd have 

```
include/
    kokkos-utils/
        concepts/
            View.hpp
tests/
    Kokkos-utils/
        concepts/
            test_View.cpp
```

An advantage would be that if the include directory is included in another project, then, within that other project, one can write `#include "kokkos-utils/concepts/View.hpp"`. 

It would be like e.g. `googletest` that does `include/gtest/...` and we include like `#include "gtest/gtest.h"`.

In the `Kokkos` family, it seems `mdspan` does it this way, but the other subprojects like `kokkos-kernels`, `kokkos-comm`,... don't.

It think it would have an impact if we include `kokkos-utils` as a git submodule. But when we go through an install,  the install dirs can probably be easily be reconfigured.

Another question is that if we start "small", we may not immediately need subfolders like `concepts`.